mollie_subscription_ept/controllers/portal.py

```diff
@@ -47,7 +45,6 @@ def portal_my_subscription(self, page=1, date_begin=None, date_end=None, sortby=
                              step=self._items_per_page)
         # content according to pager and archive selected
         orders = sale_order.search(domain, order=sort_order, limit=self._items_per_page, offset=pager['offset'])
```

Comment:
This can be changed to request.env["sale.order"] as its being used only once in the entire function